include/seeds.dao.hpp

```diff
@@ -31,17 +33,42 @@ CONTRACT dao : public contract {
           {}
 
       name alliance_scope = "alliance"_n;
-      name campaign_scope = contracts::proposals;
+      name campaign_scope = "campaign"_n;
       name milestone_scope = "milestone"_n;
-      name referendums_scope = contracts::referendums;
+      name referendums_scope = "referendum"_n;
+      name dhos_scope = "dhos"_n;
 
       std::vector<name> scopes = {
         alliance_scope,
         campaign_scope,
         milestone_scope,
-        referendums_scope
+        referendums_scope,
+        dhos_scope
       };
 
+      name alliance_fund = "alliance"_n;
+      name campaign_fund = "campaign"_n;
+      name milestone_fund = "milestone"_n;
+
+      std::vector<name> fund_types = {
+        alliance_fund,
+        campaign_fund,
+        milestone_fund
+      };
+
+      const name prop_active_size = "prop.act.sz"_n;
+      const name user_active_size = "user.act.sz"_n;
+      const name cycle_vote_power_size = "votepow.sz"_n;
+      const name dhos_vote_size = "dho.vote.sz"_n; 
+      const name linear_payout = "linear"_n;
+      const name stepped_payout = "step"_n;
+
+      typedef struct dhovote {
+        name dho;
+        uint64_t points;
```

Comment:
points here is always 0-100 right, like percentage points...

Reply:
amm well no, I am storing there the result of multiplying:
`assigned percentage * user's contribution score`
so it can be greater than 100. 

When a user votes, the user needs to call `votedhos`, in that function the contract validates that the percentage the user is assigning sums up to 100. The user only has to vote for the orgs he is going to assign a percentage bigger than 0, currently the minimum is 1% because it checks the sum is 100, I would have to modify it to sum up to 10000 if you would like to include 2 decimals.

Reply:
I don't think we need decimals, and thanks for the explanation